pkg/util/mount/mount.go

```diff
@@ -49,16 +49,20 @@ type MountPoint struct {
 	Pass   int
 }
 
-// SafeFormatAndMount probes a device to see if it is formatted. If
-// so it mounts it otherwise it formats it and mounts it
+// SafeFormatAndMount probes a device to see if it is formatted.
+// Namely it checks to see if a file system is present. If so it
+// mounts it otherwise the device is formatted first then mounted.
 type SafeFormatAndMount struct {
 	Interface
 	Runner exec.Interface
 }
 
-// Mount mounts the given disk. If the disk is not formatted and the disk is not being mounted as read only
-// it will format the disk first then mount it.
-func (mounter *SafeFormatAndMount) Mount(source string, target string, fstype string, options []string) error {
+// FormatAndMount formats the given diskm if needed, and mounts the
+// given disk. That is if the disk is not formatted and it is not
```

Comment:
redundant "given disk".

maybe more succinct as "FormatAndMount formats the given disk, if needed, before mounting."?